sanctum-solana-client-utils/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "sanctum-solana-client-utils"
+version = "0.1.0"
+edition = "2021"
+license.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+log = { workspace = true }
+medians = "3.0"
```

Comment:
i was thinking we just standardize making all dependencies workspace dependencies so that we dont ever run into the case of having different vers of the same crate